pypi_count/command_line_interface.py

```diff
@@ -6,7 +6,7 @@
 
 import typer
 
-from src.py_counter import PyCount
+from pypi_count.py_counter import PyPiCount
 
 cli = typer.Typer()
 
```

Comment:
Should `input_file` be an argument, rather than an option? Arguments are required by default and the user wouldn't need to specify the `--input-file` option to enter the path.

Reply:
If you make this change, make sure to update the `README` usage instructions.

Reply:
@solisa986